.github/workflows/benchmarks.yml

```diff
@@ -113,7 +120,7 @@ jobs:
       - run: pip install -U pip setuptools
       - run: python setup.py develop
       - run: pip install --progress-bar off -r requirements-bench.txt
-      - run: pip install --progress-bar off -U git+https://github.com/sile/kurobako-py
+      - run: pip install --progress-bar off -U git+https://github.com/c-bata/kurobako-py@fix-log-completed-trials
```

Comment:
Set the branch name of my fork to fix a kurobako-py error.
See https://github.com/CyberAgentAILab/cmaes/runs/5986187038?check_suite_focus=true for details.